recipes/metaeuk/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0-c87cf06" %}
```

Comment:
When you do tag a regular release, can you use regular versioning, such as 0.1.0?